SonarQube.Bootstrapper/ArgumentProcessor.cs

```diff
@@ -163,7 +163,7 @@ private static IBootstrapperSettings CreatePreProcessorSettings(IList<string> ch
             {
                 Debug.Assert(fileProvider.PropertiesFile != null);
                 Debug.Assert(!string.IsNullOrEmpty(fileProvider.PropertiesFile.FilePath), "Expecting the properties file path to be set");
-                childArgs.Add(string.Format(System.Globalization.CultureInfo.InvariantCulture, "{0}\"{1}\"", FilePropertyProvider.Prefix, fileProvider.PropertiesFile.FilePath));
+                childArgs.Add(string.Format(System.Globalization.CultureInfo.InvariantCulture, "{0}{1}", FilePropertyProvider.Prefix, fileProvider.PropertiesFile.FilePath));
```

Comment:
I moved all the responsibility of escaping and quoting args to the process runner. Paths do not need to be quoted apriori, because the process runner will quote all arguments